CMakeLists.txt

```diff
@@ -123,7 +123,12 @@ add_library(common STATIC ${common_srcs})
 # OpenGL
 find_package(OpenGL REQUIRED)
 find_package(GLUT REQUIRED)
-include_directories(${OPENGL_INCLUDE_DIRS} ${GLUT_INCLUDE_DIRS})
+find_package(GLM REQUIRED)
```

Comment:
Did you forget to add a CMake find file for GLM (ie FindGLM.cmake)? I don't think cmake has a default GLM find feature.

> CMake Error at CMakeLists.txt:126 (find_package):
>   By not providing "FindGLM.cmake" in CMAKE_MODULE_PATH this project has
>   asked CMake to find a package configuration file provided by "GLM", but
>   CMake did not find one.
> 
>   Could not find a package configuration file provided by "GLM" with any of
>   the following names:
> 
>     GLMConfig.cmake
>     glm-config.cmake

>   Add the installation prefix of "GLM" to CMAKE_PREFIX_PATH or set "GLM_DIR"
>   to a directory containing one of the above files.  If "GLM" provides a
>   separate development package or SDK, be sure it has been installed.